.zuul.yaml

```diff
@@ -50,28 +52,37 @@
               - ^OWNERS$
               - ^SECURITY_CONTACTS$
               - ^.gitignore$
-    cloud-provider-openstack-acceptance-test-k8s-cinder:
-      jobs:
-        - cloud-provider-openstack-acceptance-test-k8s-cinder:
-            irrelevant-files:
-              - ^docs/.*$
-              - ^.*\.md$
-              - ^OWNERS$
-              - ^SECURITY_CONTACTS$
-              - ^.gitignore$
+    # Disable the job for Kubernetes in-tree cinder function.
+    # cloud-provider-openstack-acceptance-test-k8s-cinder:
+    #   jobs:
+    #     - cloud-provider-openstack-acceptance-test-k8s-cinder:
+    #         irrelevant-files:
+    #           - ^docs/.*$
+    #           - ^.*\.md$
+    #           - ^OWNERS$
+    #           - ^SECURITY_CONTACTS$
+    #           - ^.gitignore$
     # Disable Standalone Cinder Driver Job as it is deprecated
     # cloud-provider-openstack-acceptance-test-standalone-cinder:
-    #  jobs:
-    #    - cloud-provider-openstack-acceptance-test-standalone-cinder:
-    #        irrelevant-files:
-    #          - ^docs/.*$
-    #          - ^.*\.md$
-    #          - ^OWNERS$
-    #          - ^SECURITY_CONTACTS$
-    #          - ^.gitignore$
+    #   jobs:
+    #     - cloud-provider-openstack-acceptance-test-standalone-cinder:
+    #         irrelevant-files:
+    #           - ^docs/.*$
+    #           - ^.*\.md$
+    #           - ^OWNERS$
+    #           - ^SECURITY_CONTACTS$
+    #           - ^.gitignore$
     cloud-provider-openstack-acceptance-test-csi-cinder:
       jobs:
         - cloud-provider-openstack-acceptance-test-csi-cinder:
+            files:
+              - cmd/cinder-csi-plugin/.*
```

Comment:
pkg/cloudprovider/providers/openstack is also needed

Reply:
I can't find it's used, cinder-csi has a copy of openstack package.

Reply:
ya, inside that its used :) , https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/openstack/openstack.go#L30

Reply:
such a bad design :-(

Reply:
ya, refactor is necessary